swan-lake/development-tutorials/source-code-dependencies/customizing-formatting.md

```diff
@@ -0,0 +1,388 @@
+---
+title: Customization of formatting
+description: The section on providing custom configurations to the formatter.
+keywords: ballerina, programming language, formatting, custom configurations
+permalink: /learn/customizing-formatting/
+active: customizing-formatting
+intro: This feature allows developers to customize the formatting of Ballerina projects. This is done with the use of a configuration file. This configuration file can be a local or a remote file, which allows for consistency in code style across projects and simplifies the process of enforcing formatting standards. This is introduced as an experimental feature in Ballerina 2201.9.0.
+---
+
+## Configuring
+
+Users can provide custom formatting configurations with the use of a separate configuration file of the `toml` format. Users can provide an arbitrary name for the file.
+
+The formatting configuration file can be specified in two ways:
```

Comment:
This says can be specified in two ways, but there are 4 bullet points given below which is confusing. So first list the two ways clearly and then mention additional points after another sentence etc.